Microsoft.Azure.Devices.LongHaul.Module/LoggingConstants.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.Devices.LongHaul.Module
+{
+    internal static class LoggingConstants
+    {
+        // Metrics
+
+        public const string DisconnectedDurationSeconds = "DisconnectedDurationSeconds";
+        public const string TotalTelemetryMessagesSent = "TotalTelemetryMessagesSent";
+        public const string TelemetryMessageDelaySeconds = "TelemetryMessageDelaySeconds";
+        public const string MessageBacklog = "MessageBacklog";
+        public const string C2mDirectMethodDelaySeconds = "C2mDirectMethodDelaySeconds";
+        public const string TotalTwinUpdatesReported = "TotalTwinUpdatesReported";
+        public const string TotalTwinCallbacksHandled = "TotalTwinCallbacksHandled";
+        public const string TotalDesiredPropertiesHandled = "TotalDesiredPropertiesHandled";
+        public const string TotalC2mMessagesCompleted = "TotalC2mMessagesCompleted";
+        public const string TotalC2mMessagesRejected = "TotalC2mMessagesRejected";
+        public const string C2mMessageOperationSeconds = "C2dMessageOperationSeconds";
+        public const string ReportedTwinUpdateOperationSeconds = "ReportedTwinUpdateOperationSeconds";
+        public const string FileUploadOperationSeconds = "FileUploadOperationSeconds";
+
+        // Events
+
+        public const string StartingRun = "StartingRun";
+        public const string ConnectedEvent = "Connected";
+        public const string DiscconnectedEvent = "Disconnected";
+        public const string EndingRun = "EndingRun";
+
+        // Logging properties
+
+        public const string TestClient = "testClient";
+        public const string RunId = "runId";
+        public const string SdkLanguage = "sdkLanguage";
+        public const string SdkVersion = "sdkVersion";
+        public const string OperationName = "OperationName";
```

Comment:
We should use camelCase for logging properties.
```suggestion
        public const string OperationName = "operationName";
```